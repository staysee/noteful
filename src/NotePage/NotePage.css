.NotePage {
    padding: 10px;
}

.NotePage__content {
    max-width: 380px;
    margin-left: 8px;
    padding: 10px;
    color: #f0fbff
}