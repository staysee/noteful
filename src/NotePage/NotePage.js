import React from 'react'
import '/NotePage.css'

function NotePage(props){
    return (
        <div className="NotePage">
            <Note />

            <div className="NotePage__content">
                {props.content}
            </div>
        </div>
    )
}

export default NotePage