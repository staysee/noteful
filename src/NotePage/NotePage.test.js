import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import NotePage from './NotePage'

describe(`NotePage component`, () => {

  it('renders a .NotePage by default', () => {
    const wrapper = shallow(<NotePage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders a Note with note prop', () => {
    const props = {
      match: {
        params: {
          noteId: 'cbc787a0-ffaf-11e8-8eb2-f2801f1b9fd1'
        }
      }
    }

    const context = {
      notes: [{
        id: `cbc787a0-ffaf-11e8-8eb2-f2801f1b9fd1`,
        name: `Dogs`,
        modified: `2019-01-03T00:00:00.000Z`,
        // folderId: b0715efe-ffaf-11e8-8eb2-f2801f1b9fd1,
        content: "Corporis accusamus placeat.\n \rUnde."
      }]
    }

    const note = shallow(<NotePage {...props} />, context)
      .find('Note')
    expect(toJson(note)).toMatchSnapshot()
  })
})