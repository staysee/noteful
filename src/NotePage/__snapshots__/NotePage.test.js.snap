// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotePage component renders a .NotePage by default 1`] = `
<div
  className="NotePage"
>
  <Note
    onDeleteNote={[Function]}
  />
  <div
    className="NotePage__content"
  />
</div>
`;

exports[`NotePage component renders a Note with note prop 1`] = `
<Note
  onDeleteNote={[Function]}
/>
`;
